cmake/dependencies/AMReX.cmake

```diff
@@ -87,9 +87,13 @@ macro(find_amrex)
         endif()
 
         # shared libs, i.e. for Python bindings, need relocatable code
-        #   openPMD: currently triggers shared libs (TODO)
-        if(WarpX_PYTHON OR (WarpX_LIB AND BUILD_SHARED_LIBS) OR ABLASTR_POSITION_INDEPENDENT_CODE OR WarpX_OPENPMD)
-            set(AMReX_PIC ON CACHE INTERNAL "")
+        if(WarpX_PYTHON OR
```

Comment:
Since `ABLASTR_POSITION_INDEPENDENT_CODE` defaults to `ON`, doesn't this still always use `-fPIC` by default?

Reply:
Right, sorry, I misread this and was thinking that this PR was turning off the PIC. It is instead turning off the shared lib. I can approve.